tweeli/config/config.ini_example

```diff
@@ -1,9 +0,0 @@
-[api]
```

Comment:
There is a reason for naming this file config.ini_example:
People may accidentally push their keys to the repo! That's why we decided to name it  _example 😉

Reply:
And just imagine they sends their data, what happen? Ha? Github will sends a notification for them. Its 2018 its not like 1990.